### DISCUSSION TITLE
Regional geoid determination

### DISCUSSION THREAD

**Discussion by gnssgrid** [groops-devs/groops#249] [General]
Hello
I want to try GROOPS for regional geoid modelling, but din't find any example with input and output files. Do you have some more detail tutorial with sample data?
Regards
Dzevad

**Comment by tmayerguerr** [comment]
Hi Dzevad,

The problem is that gravity field data is not always freely available and I am not allowed to simply publish it.
However, I can give you my scripts ([DACH_geoid.zip](https://github.com/groops-devs/groops/files/14849739/DACH_geoid.zip)) without data for calculating the geoid from the [Cookbook example](https://groops-devs.github.io/groops/html/cookbook.regionalGeoid.html). Please note that this example is very complex and only works on computers with a lot of memory.

regards
Torsten

**Comment by gnssgrid** [comment]
Hi Torsten

I understand. Well, I will try your scripts. I already download SRTM DEM (3") and create some random points and then calculate gravity using EGM2008. Do you have more detailed explanation of input gravity and GNSS/lev point files. I see that input gravity point file has  L B H grav factor accuracy factor. H is ellipsoidal? What is factor? Also I don't have accuracy of my gravity data (suppose that it is used for weight)? In GNSS/lev file there is four columns L B H normh. H is ellipsoidal and normh is probably orthometric (normal) height?
Scripts 020 and 030 finished successfully.

Best regards
Dzevad Krdzalic
[data.zip](https://github.com/groops-devs/groops/files/14857973/data.zip)
